package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
`yarn lint` locally will still catch it, but `lint-staged` seems to have an issue with the way it runs commands such that the only way for ESLint to work on `.storybook` is the `--no-ignore` flag you added. But then you end up linting code that _shouldn't_ be linted too. Can't win haha! Not high priority, but if we could move storybook's config to a non-`.`-prefixed directory as you suggested, that would work.